examples/reference/indicators/Tqdm.ipynb

```diff
@@ -0,0 +1,177 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import time\n",
+    "\n",
+    "import pandas as pd\n",
+    "import numpy as np\n",
+    "\n",
+    "import panel as pn\n",
+    "from panel.widgets.tqdm_indicator import Tqdm\n",
+    "\n",
+    "pn.extension()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Tqdm Indicator\n",
```

Comment:
No titles in reference notebooks.

Reply:
Ok. I have implemented it. 

But have you thought about how Screenshots or Videos of a Reference Guide without a title communicate? It can be a little bit difficult for the casual viewer to see what this is about?

Reply:
Reference Guides are primarily documentation so they should be optimized for rendering on the website which adds the title and a user that opens the notebook file can see the notebook title. For a screenshot or video it's easy enough to temporarily insert a title.